exosys/Cargo.lock

```diff
@@ -2199,7 +2203,7 @@ dependencies = [
  "anyhow",
  "async-stream",
  "clap 3.2.14",
- "definitions",
+ "definitions 0.1.0 (git+https://github.com/varovainen/parity-signer.git?branch=va-2022-07-28-parser)",
```

Comment:
Careful with these!

Reply:
Sure, it will go away as soon as "substrate-parser" crate is in production

Reply:
Should be fixed now